crates/bindings-macro/Cargo.toml

```diff
@@ -13,6 +13,7 @@ bench = false
 
 [dependencies]
 spacetimedb-primitives.workspace = true
+spacetimedb-sql-parser.workspace = true
```

Comment:
We have to parse on the server regardless, so I would say let's not add it to the bindings.